Extension/src/LanguageServer/utils.ts

```diff
@@ -30,6 +31,17 @@ export function rangeEquals(range1: vscode.Range | Range, range2: vscode.Range |
     range1.end.line === range2.end.line && range1.end.character === range2.end.character;
 }
 
+export function getFileFromPath(filePath: string): string | undefined {
```

Comment:
It seems like this should return "string" and never undefined, i.e. return filePath on error?

Reply:
Could you give a code snippet?